front/migrations/20240829_backfill_keys_without_group_id.ts

```diff
@@ -0,0 +1,66 @@
+import type { LightWorkspaceType } from "@dust-tt/types";
+
+import { Authenticator } from "@app/lib/auth";
+import { Workspace } from "@app/lib/models/workspace";
+import { GroupResource } from "@app/lib/resources/group_resource";
+import { KeyResource } from "@app/lib/resources/key_resource";
+import { KeyModel } from "@app/lib/resources/storage/models/keys";
+import { renderLightWorkspaceType } from "@app/lib/workspace";
+import type { Logger } from "@app/logger/logger";
+import { makeScript } from "@app/scripts/helpers";
+
+async function backfillApiKeys(
+  workspace: LightWorkspaceType,
+  logger: Logger,
+  execute: boolean
+) {
+  logger.info("Handle workspace " + workspace.id);
+  const auth = await Authenticator.internalAdminForWorkspace(workspace.sId);
+
+  if (execute) {
+    const systemGroup = await GroupResource.fetchWorkspaceSystemGroup(auth);
+    if (systemGroup.isOk()) {
+      await KeyResource.model.update(
+        { groupId: systemGroup.value.id },
+        {
+          where: {
+            workspaceId: workspace.id,
+            isSystem: true,
+          },
+        }
+      );
+    }
+  }
+  logger.info("Done with workspace " + workspace.id);
```

Comment:
Just in case let's log the error of `systemGroup`. If those keys have missing system group.